schema_based_on_diagram.sql

```diff
@@ -30,4 +30,25 @@ ALTER TABLE
 CREATE TABLE "species"(
     "id" BIGINT NOT NULL,
     "name" VARCHAR(255) NOT NULL
-);
\ No newline at end of file
+);
+ALTER TABLE
+    "species" ADD PRIMARY KEY("id");
+CREATE TABLE "owners"(
+    "id" INTEGER NOT NULL,
+    "full_name" VARCHAR(255) NOT NULL,
+    "age" INTEGER NOT NULL
+);
+ALTER TABLE
+    "owners" ADD PRIMARY KEY("id");
+ALTER TABLE
+    "specializations" ADD CONSTRAINT "specializations_species_id_foreign" FOREIGN KEY("species_Id") REFERENCES "species"("id");
+ALTER TABLE
+    "visits" ADD CONSTRAINT "visits_vet_id_foreign" FOREIGN KEY("vet_id") REFERENCES "vets"("id");
+ALTER TABLE
+    "specializations" ADD CONSTRAINT "specializations_vet_id_foreign" FOREIGN KEY("vet_id") REFERENCES "vets"("id");
+ALTER TABLE
+    "animals" ADD CONSTRAINT "animals_species_id_foreign" FOREIGN KEY("species_id") REFERENCES "species"("id");
+ALTER TABLE
+    "visits" ADD CONSTRAINT "visits_animal_id_foreign" FOREIGN KEY("animal_id") REFERENCES "animals"("id");
+ALTER TABLE
+    "animals" ADD CONSTRAINT "animals_owner_id_foreign" FOREIGN KEY("owner_id") REFERENCES "owners"("id");
```

Comment:
- [ ] Hi @Team, like the previous reviewer mentioned, please make sure you are implementing this requirement according to the attached diagram (ERD Design). It is a major requirement. Please be reminded that this is no longer the prior vets clinic. Looking at the diagram, you will see entities like `patients`, `medical_histories`, `invoices`, etc. which is not what you have here currently.
- [ ] Kindly implement the required features and submit them again.

- An example with creating a `Patient` table would be:
```
CREATE TABLE patients (
  id SERIAL PRIMARY KEY,
  name VARCHAR(100) ,
  date_of_birth DATE ,
);
```